layers/meta-balena-isg-503/recipes-bsp/u-boot/u-boot-rockchip.inc

```diff
@@ -201,7 +201,7 @@ do_install () {
 
 }
 
-FILES_${PN} = "/boot ${sysconfdir}"
+FILES:${PN} = "/boot ${sysconfdir}"
```

Comment:
This commit can be squashed in this other commit: https://github.com/balena-os/balena-isg/pull/239/commits/76ed311c4dec53e4e0e7ed4f2604184033416e74